e2e/code-generator/java.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+export sourcefile=$1
+# exit if no source file is specified
+if [ -z "$sourcefile" ]
+then
+  echo "no source file is specified"
+  exit 1
+fi
+
+mv ${sourcefile} Main.java
+javac Main.java
```

Comment:
Maybe we can use `java Main.java` 
![image](https://github.com/LinuxSuRen/api-testing/assets/77964041/56e4f0a3-e9cf-4b5f-a3e5-a95c9d5bd549)
the effect is equivalent to the above command.

Reply:
Learned. It's cool.